.github/workflows/qa-testbench.yaml

```diff
@@ -0,0 +1,125 @@
+name: QA Testbench run
+
+on:
+  workflow_dispatch: {}
+  schedule:
+  # * is a special character in YAML so you have to quote this string
+    - cron:  '0 0 * * *'
+  
+inputs:
+  generation:
+    description: 'Specifies the generation template which should be used by the testbench run'
+    required: false
+    default: 'Zeebe SNAPSHOT'
+  branch:
+    description: 'Specifies the branch, for which the QA Testbench run should be executed'
+    default: 'main'
+    required: false
+  
+jobs:     
+  qa-testbench:
+    name: QA Testbench run
+    runs-on: ubuntu-latest
+    env:
+      IMAGE: "gcr.io/zeebe-io/zeebe"
+      GENERATION_TEMPLATE: "${{ github.event.inputs.generation }}"
+      BRANCH_NAME: "${{ github.event.inputs.branch }}"
+      ZEEBE_AUTHORIZATION_SERVER_URL: "https://login.cloud.ultrawombat.com/oauth/token"
+      BUILD_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+    steps:
+      # Dynamic environment variables are not supported by GHA
+      # https://brandur.org/fragments/github-actions-env-vars-in-env-vars
+      - id: set-env
+        name: Set environment variables
+        run: |
+          echo 'TAG=SNAPSHOT-${GITHUB_SHA::8}' >> $GITHUB_ENV
+          echo 'QA_RUN_VARIABLES="{\"zeebeImage\": \"$IMAGE:SNAPSHOT-${GITHUB_SHA::8}\", \"generationTemplate\": \"$GENERATION_TEMPLATE\", "\"channel\": \"Internal Dev\", \"branch\": \"$BRANCH_NAME\", \"build\": \"$BUILD_URL\", \"businessKey\": \"$BUILD_URL\", \"processId\": \"qa-protocol\"}"' >> $GITHUB_ENV
+          echo 'BUSINESS_KEY=$BUILD_URL' >> $GITHUB_ENV
+                                    
+      - uses: actions/checkout@v3
+        with:
+          ref: "${{ github.event.inputs.branch }}"
+      - name: Import Secrets
+        id: secrets
+        uses: hashicorp/vault-action@v2.4.1
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+          secrets: |
+            secret/data/common/ci-zeebe/jenkins ZEEBE_GCR_SERVICEACCOUNT_JSON;
+            secret/data/common/ci-zeebe/testbench-secrets-1.x-prod clientSecret;
+            secret/data/common/ci-zeebe/testbench-secrets-1.x-prod contactPoint;
+      - uses: actions/setup-java@v3.3.0
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+      - uses: actions/setup-go@v3
+        with:
+          go-version: "1.15"
+      - name: Build Go
+        run: ./build.sh
+        working-directory: clients/go/cmd/zbctl
+      - name: Package Zeebe
+        run: mvn -B -DskipTests -DskipChecks package
+        
+      - name: Login to GCR
+        uses: docker/login-action@v2
+        with:
+          registry: gcr.io
+          username: _json_key
+          password: ${{ steps.secrets.outputs.GCR_JSON_KEY }}
```

Comment:
:x: I think this should be `ZEEBE_GCR_SERVICEACCOUNT_JSON`, based on the vault action above.

Reply:
Thanks good catch!